RefactoringEssentials/CSharp/CodeRefactorings/Synced/ConvertExpressionBodyToStatementBodyCodeRefactoringProvider.cs

```diff
@@ -66,6 +66,14 @@ static void HandleMethodCase(CodeRefactoringContext context, SyntaxNode root, Sy
             ExpressionSyntax expr;
             if (!IsExpressionBody(method.Body, method.ExpressionBody, out expr))
                 return;
+            
+            var returnType = method.ReturnType as PredefinedTypeSyntax;
+            if (returnType == null)
```

Comment:
What if the ReturnType is not predefined? Do we want to hide the refactoring in those cases?